docs/getting-started.md

```diff
@@ -30,96 +32,38 @@ Additional resources
 version and other examples in [Sonatype Central
-Repository](https://search.maven.org/artifact/com.expediagroup/graphql-kotlin-schema-generator)
+Repository](https://search.maven.org/artifact/com.expediagroup/graphql-kotlin-spring-server)
 
 ### Maven
 
 ```xml
 <dependency>
   <groupId>com.expediagroup</groupId>
-  <artifactId>graphql-kotlin-schema-generator</artifactId>
+  <artifactId>graphql-kotlin-spring-server</artifactId>
   <version>${latestVersion}</version>
 </dependency>
 ```
 
 ### Gradle
 
 ```groovy
-compile(group: 'com.expediagroup', name: 'graphql-kotlin-schema-generator', version: "$latestVersion")
-```
-
-## Generating a schema
-
-`graphql-kotlin-schema-generator` provides a single function, `toSchema`, to generate a schema from Kotlin objects.
-
-```kotlin
-class Query {
-  fun getNumber() = 1
-}
-
-val config = SchemaGeneratorConfig(listOf("com.expediagroup"))
-val queries = listOf(TopLevelObject(Query()))
-val schema: GraphQLSchema = toSchema(config = config, queries = queries)
-```
-
-generates a `GraphQLSchema` with IDL that looks like this:
-
-```graphql
-schema {
-  query: Query
-}
-
-type Query {
-  getNumber: Int!
-}
+compile(group: 'com.expediagroup', name: 'graphql-kotlin-spring-server', version: "$latestVersion")
 ```
 
-The `GraphQLSchema` generated can be used to expose a GraphQL API endpoint.
-
-### `toSchema`
+## Running a Server
+`graphql-kotlin-spring-server` is a combination schema generator and the server libraries. If you are looking to run a GraphQL server, this is the place to start.
 
-This function accepts four arguments: `config`, `queries`, `mutations` and `subscriptions`. The `queries`, `mutations`
-and `subscriptions` are a list of `TopLevelObject`s and will be used to generate corresponding GraphQL root types. See
-below on why we use this wrapper class. The `config` contains all the extra information you need to pass, including
-custom hooks, supported packages, and name overrides. See the full documentation: [Spring Server Overview](spring-server/spring-overview).
+See the docs in [Spring Server Overview](./spring-server/spring-overview.md).
 
-You can see the definition for `toSchema` [in the
-source](https://github.com/ExpediaGroup/graphql-kotlin/blob/master/graphql-kotlin-schema-generator/src/main/kotlin/com/expediagroup/graphql/toSchema.kt)
+## Creating a Client
+`graphql-kotlin-plugins` can be used to generate a `graphql-kotlin-client` from an existing schema that is easy to use and type-safe.
 
-### Class `TopLevelObject`
+See the docs in [Client Overview](./client/client-overview.md).
 
-`toSchema` uses Kotlin reflection to build a GraphQL schema from given classes using `graphql-java`'s schema builder. We
-don't just pass a `KClass` though, we have to actually pass an object, because the functions on the object are
-transformed into the data fetchers. In most cases, a `TopLevelObject` can be constructed with just an object:
+## Generating a Schema
```

Comment:
I think we should move this up -> i.e. start with generating a schema, then spring server (as it references the schema generation) and close with the client (which runs against a schema).